src/commands/Minion/fish.ts

```diff
@@ -122,7 +122,21 @@ export default class extends BotCommand {
 			);
 		}
 
-		const maxTripLength = msg.author.maxTripLength('Fishing');
+		let maxTripLength = msg.author.maxTripLength('Fishing');
+		const tackleBoxes = [
+			"Champion's tackle box",
+			'Professional tackle box',
+			'Standard tackle box',
+			'Basic tackle box'
+		];
+		for (let i = 0; i < tackleBoxes.length; i++) {
+			if (msg.author.hasItemEquippedOrInBank(tackleBoxes[i])) {
+				let num = Time.Minute * (tackleBoxes.length - i);
```

Comment:
One minute per tier seems a little low, fish sack is +9 minutes for a tradeable item, I think 2 minutes per tier might be a little better, without being OP.